metricflow/model/validations/unique_valid_name.py

```diff
@@ -35,6 +35,7 @@ class MetricFlowReservedKeywords(enum.Enum):
     """Enumeration of reserved keywords with helper for accessing the reason they are reserved"""
 
     METRIC_TIME = "metric_time"
+    MF_INTERNAL_UUID = "mf_internal_uuid"
```

Comment:
Oh I see, this is because we're adding this under the covers, right?

Shall we make the prefix `metricflow` instead of `mf`? I'm ok either way, `mf_internal_` is already pretty distinctive.